src/components/SingleName/SubDomains.js

```diff
@@ -103,6 +105,18 @@ function SubDomains({
   ...rest
 }) {
   const { t } = useTranslation()
+  const { data: { wildcardResolverDomain } = {} } = useQuery(
+    WILDCARD_RESOLVER_DOMAIN,
+    {
+      variables: {
+        name: domain.name
+      }
+    }
+  )
+
+  const isWildcard =
```

Comment:
This is hard to follow. Would it be possible to rename this to something like `isInRegistry`?

Reply:
I will get rid of this because the logic now moved as part of `wildcardResolverDomain`. However, I just realised that there is a different issue. `This domain may have other subdomains that cannot be shown as they are off chain` message can be displayed at `/name/1.offchainexample.eth/subdomains` but can NOT be displayed at `/name/offchainexample.eth/subdomains` because there is currently no way to tell that the parent domain may have wildcard subdomains.

Reply:
We should show that message for any domain which a) doesn't exist in the registry, or b) has a resolver that implements `resolve`. If the resolver exists and *doesn't* implement `resolve`, we know there can't be wildcard subdomains.

RE unicode support, are we changing to use Ethers in this PR?

Reply:
` ` now indicates wildcard subdomain and its parent https://github.com/ensdomains/ui/pull/160/commits/56e9df7dcba8146559bd75e5647a5185396967a6 .

Reply:
```We should show that message for any domain which a) doesn't exist in the registry, or b) has a resolver that implements resolve. If the resolver exists and doesn't implement resolve, we know there can't be wildcard subdomains.```

Actually I am a bit confused now. I initially checked whether the resolver has `resolve` function. I remove the check because you suggested not to rely on `resolve` function but now we do rely on?